docs/src/test-advanced.md

```diff
@@ -200,6 +201,57 @@ export const test = base.extend<{ saveLogs: void }>({
 });
 ```
 
+## Launching a development web server during the tests
+
+To launch a web server during the tests, use the `webServer` option in the [configuration file](#configuration-object).
+
+This will automatically launch and wait for the web server to be ready before the tests will start:
```

Comment:
Please explain the baseURL connection here, and give an example test that uses it.